src/bosh_azure_cpi/spec/unit/models/vm_cloud_props_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'unit/cloud/shared_stuff.rb'
+
+describe Bosh::AzureCloud::VMCloudProps do
+  include_context 'shared stuff'
+  describe '#initialize' do
+    context 'when availability_set is a hash' do
```

Comment:
Add one case for "when availability_set is a string".
Same for load_balancer

Reply:
it's covered in other case. 
because we track the 100% coverage.
the 
#####
else
        Bosh::AzureCloud::LoadBalancerConfig.new(
          global_azure_config.resource_group_name,
          vm_properties[LOAD_BALANCER_KEY]
        )
end
#####
must be already go through now.